test-app/tests/integration/components/headless-form-test.gts

```diff
@@ -177,7 +131,7 @@ module('Integration Component headless-form', function (hooks) {
 
   module('field.input', function () {
     test('field yields input component', async function (assert) {
-      const data = { firstName: 'Simon' };
+      const data: { firstName?: string } = {};
```

Comment:
Question: why is firstName optional here? On line 253 it is not. Just curious what the difference is.

Reply:
oh NVM I see that it's an empty object 🤦